### PR TITLE
WMS GetCapabilities Mix/MaxDenominators for layer groups [GEOS-7333]

### PR DESCRIPTION
**Pull request by NielsCharlier**
https://osgeo-org.atlassian.net/browse/GEOS-7333

### CODE REVIEW

File: src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesScaleDenominatorTest.java [hunk 1]
```diff
@@ -0,0 +1,237 @@
+/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2013 OpenPlans
```
Comment (jodygarnett) [inline src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesScaleDenominatorTest.java]: Is this a new file? Or are you reusing existing code (and thus need the open plans reference)

**Comment by jodygarnett** [comment]
This seems clear, thanks for test case.

File: src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesScaleDenominatorTest.java (hunk 1 above)
Comment (NielsCharlier) [inline src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesScaleDenominatorTest.java]: largely copied from GetCapabilitiesScaleHintTest

**Comment by NielsCharlier** [comment]
So this can be merged?

**Comment by jodygarnett** [comment]
Yep, you answered my question at least :)

**Pull request merged**